types/params.go

```diff
@@ -75,13 +76,21 @@ type VersionParams struct {
 	AppVersion uint64 `json:"app_version"`
 }
 
+// SynchronyParams influence the validity of block timestamps.
+// TODO (@wbanfield): add link to proposer-based timestamp spec when completed.
+type SynchronyParams struct {
+	Precision    time.Duration `json:"precision"`
+	MessageDelay time.Duration `json:"message_delay"`
```

Comment:
```suggestion
	Precision    time.Duration `json:"precision,string"`
	MessageDelay time.Duration `json:"message_delay,string"`
```

Reply:
This makes sense, although it looks like our rpc endpoints already give us these back as quoted?

Reply:
Yes, but that relies on the custom JSON marshaler. This allows it to work for the standard one too.